core/src/main/scala/scala/collection/parallel/mutable/ParHashSet.scala

```diff
@@ -162,7 +162,7 @@ with scala.collection.mutable.FlatHashTable.HashUtils[T] {
     // TODO parallelize by keeping separate size maps and merging them
     val tbl = new FlatHashTable[T] {
       sizeMapInit(table.length)
-      seedvalue = ParHashSetCombiner.this.seedvalue
+      seedvalue = seed
```

Comment:
`this.seedvalue = ParHashSetCombiner.this.seedvalue` instead of renaming, noting that the new name doesn't distinguish them anyway.

Reply:
So there would be another `this.seedvalue` at L187.

Reply:
good call. including this in #118